ui/src/App.vue

```diff
@@ -25,24 +27,39 @@ export default {
   },
   data() {
     return {
-      isRemote: {},
-      employmentTypes: {},
-      locations: {},
-      skills: {}
+      isRemote: [],
+      employmentTypes: [],
+      locations: [],
+      skills: []
     }
   },
-  methods: {
-    onSubmit() {
-      axios.get(mockAPI)
+  mounted() {
+    axios.get(mockAPI)
         .then(response => {
-          console.log(response);
-          // this.isRemote = response.isRemote;
+          this.isRemote = response.data.is_remote.buckets;
+          this.employmentTypes = response.data.employment_type.buckets;
+          this.locations = response.data.location.buckets;
+          this.skills = response.data.skills.buckets;
         })
-    }
+  },
+  methods: {
+    // onSubmit() {
+    //   axios.get(mockAPI)
+    //     .then(response => {
+    //       console.log(response.data);
+    //       this.isRemote = response.data.is_remote.buckets;
+    //       this.employmentTypes = response.data.employment_type;
+    //       this.locations = response.data.location.buckets;
+    //       this.skills = response.data.skills;
+    //       console.log(this.locations.buckets);
+    //     })
+    // }
   }
 }
 </script>
 
-<style>
-
-</style>
+<style scoped>
+  .header {
+    margin: 20px !important;
```

Comment:
Nooooooo try to avoid using `!important` as much as possible - maintaining css will very quickly get out of control

Reply:
Removing !important.